ya-runtime-erigon/src/main.rs

```diff
@@ -246,3 +279,23 @@ fn generate_password_file(
         .stderr(Stdio::null())
         .spawn()
 }
+
+fn parse_json(config: Option<&String>, default_chain_id: &String) -> String {
```

Comment:
`parse_json` is not very informative. How about `get_chain_from_config`? Also, having `default_chain_id` as param is odd. Better use a module-level constant.